lib/plugins/aws/invokeLocal/index.js

```diff
@@ -548,7 +548,7 @@ class AwsInvokeLocal {
         errorResult = {
           errorMessage: err.message,
           errorType: err.constructor.name,
-          stackTrace: err.stack.split('\n'),
+          stackTrace: err.stack ? err.stack.split('\n') : [],
```

Comment:
I guess it'll be more to the point to do `err.stack && err.stack.split('\n')`.

(If there's no _stack_ I would reference it with `undefined` or `null`, not an empty array, which logically suggests there is a stack but is empty)